Sources/App/Models/EventReg/EventReg.generated.swift

```diff
@@ -33,6 +34,7 @@ extension EventReg {
     init(_ string: String) {
       switch string {
       case "approved": self = .approved
+      case "notAppeared": self = .notAppeared
       case "rejected": self = .rejected
       case "waiting": self = .waiting
       default: self = .canceled
```

Comment:
Let's avoid using camel case in strings especially for strings which are stored in database. In general my suggestion: snake case for strings, camel case for vars. And maybe use `missed` for `notAppeared` and `pending` (common case) for `waiting` (if `waiting` means that admin hasn't approved it yet).